From 4063b6c356b84421a834a51306d3b0b5d9688f40 Mon Sep 17 00:00:00 2001
From: Daniel Leung <daniel.leung@intel.com>
Date: Thu, 19 Apr 2012 16:11:01 -0700
Subject: Defined TARGET_BOARD_PLATFORM for full_x86 target

The pc_std target and associated projects depend on
TARGET_BOARD_PLATFORM to associate with property
<ro.board.platform>. This patch adds this into
full_x86 so these projects can also be built
with full_x86 target.

Issue: AIA-47
Change-Id: I49421d848ca011df17f8de463b157beb0b1d090b
Signed-off-by: Daniel Leung <daniel.leung@intel.com>
---
 target/board/generic_x86/BoardConfig.mk |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/target/board/generic_x86/BoardConfig.mk b/target/board/generic_x86/BoardConfig.mk
index 4a9e5fe..a75e29b 100644
--- a/target/board/generic_x86/BoardConfig.mk
+++ b/target/board/generic_x86/BoardConfig.mk
@@ -33,3 +33,6 @@ BUILD_EMULATOR_OPENGL := true
 # Build and enable the OpenGL ES View renderer. When running on the emulator,
 # the GLES renderer disables itself if host GL acceleration isn't available.
 USE_OPENGL_RENDERER := true
+
+# For <ro.board.platform>
+TARGET_BOARD_PLATFORM := full_x86
-- 
1.7.4.1

